### PR TITLE
feat: add file.File for binary files and update other files formats

### PR DESCRIPTION
**Pull request by Lee-000**

### CODE REVIEW

File: standard/file/File.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  fields:
+    - name: key
+      type: string
+
+    - name: format
+      type: string
+      nullable: True
+
+    - name: size
+      type: int32
```
Comment (linjiX) [inline standard/file/File.yaml]: `int32` or `int64`?

File: standard/file/File.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+type: template
+parameters:
+  - name: metadata
+    default: null
```
Comment (linjiX) [inline standard/file/File.yaml:5]: `null` or `[]`

Comment (linjiX) [reply standard/file/File.yaml:5]: use graviti.portex to run your code first

Comment (Lee-000) [reply standard/file/File.yaml:5]: `null` is ok, it will be treated as empty fields.

File: standard/file/File.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+type: template
+parameters:
+  - name: metadata
```
Comment (linjiX) [inline standard/file/File.yaml]: do we have a better name for `metadata`?

**Review by linjiX — APPROVED** [review]

**Pull request merged**